src/__tests__/TicketMap/index.test.tsx

```diff
@@ -19,19 +46,18 @@ const createTouchEvent = (target: HTMLElement, pageX = 0, pageY = 0) => ({
 });
 
 describe("TicketMap", () => {
-  let wrapper: ShallowWrapper<Props, State, TicketMap>;
+  let wrapper: ReactWrapper<Props, State, TicketMap>;
```

Comment:
❓  Why did we have to use ReactWrapper here instead of Shallow?

Reply:
I had to change the test to start using `mount` instead of `shallow` beacuse the `ref` only works with `mount` (https://github.com/enzymejs/enzyme/issues/1394#issuecomment-348414481) 
The `TicketMap` component is using `ref` to link the `div` element to the svg fetched from S3